src/Engine/EngineBase.php

```diff
@@ -29,4 +35,18 @@ public function checkImageUrl(string $imageUrl): void
             throw new OcrException('image-url-error', ['https://upload.wikimedia.org']);
         }
     }
+
+    /**
+     * @param string[]|null $langs
+     * @throws OcrException
+     */
+    public function validateLangs(?array $langs): void
+    {
+        $intuition = new Intuition();
```

Comment:
The Intuition service should be injected instead (which is what I've done in #24, so if that lands first you can just use `$this->getIntuition()`).

Reply:
Good point. Thank you!

Reply:
I injected `Intuition` in the `EngineBase` class constructor, we can review and consolidate the best approach.